<launch>
  <node name="marble_simulator_ode_node" pkg="marble_simulator_ode" type="marble_simulator_ode_node">
    <param name="obstacle_file" value="$(find marble_simulator_ode)/configs/c0000.json"/>
    <param name="texture_path" value="$(find marble_simulator_ode)/textures/"/>
    <param name="time_step" value="0.005"/>
  </node>
</launch>